src/database/schema/ChatSchema.ts

```diff
@@ -240,7 +266,9 @@ class ChatSchema implements BaseDbSchema {
     userId: string,
     channelId: string,
     message: string,
-    localDb: SQLiteDatabase
+    localDb: SQLiteDatabase,
+    type: 'regular' | 'system' = 'regular',
+    status: 'pending' | 'sent' = 'pending'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The parameters `localDb`, `type`, and `status` have been added to the function signature. Ensure that all calls to this function are updated to include these parameters. Also, consider updating the function documentation to reflect these changes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->